sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -3300,7 +3297,7 @@ object Radar {
         RadarNotificationHelper.showNotifications(context, events)
 
         for (event in events) {
-            logger.i("📍 Radar event received | type = ${RadarEvent.stringForType(event.type)}; link = https://radar.com/dashboard/events/${event._id}")
+            logger.i("📍 Radar event received | type = ${RadarEvent.stringForType(event.type)} | replayed = ${event.replayed} link = https://radar.com/dashboard/events/${event._id}")
```

Comment:
think we need the `|` to be a `;` in between `type` and `replayed` and then a `;` between `replayed` and `link`

Reply:
Ah yes, just saw the other ones, changed: https://github.com/radarlabs/radar-sdk-android/pull/312/commits/cb49d1a595a4fc2a929b603609c19e9887d218d1